### PR TITLE
added in feature branch

### PR DESCRIPTION
**Pull request by jaimin45**
Add some file from feature/api-design in Develop branch.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "nodemon server.js --exec babel-node --presets babel-preset-env",
+    "lint": "eslint src/js"
+  },
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "body-parser": "^1.19.0",
+    "express": "^4.17.1",
+    "nodemon": "^2.0.4",
```
Comment (kapilbarad) [inline package.json:16]: `nodemon` should be in devDependencies:
```
npm i nodemon --save-dev
```


CC: @jaimin45 ashish

File: README.md [hunk 2]
```diff
@@ -1 +1,27 @@
...
+simple project with node and express to create restful api's (CRUD)
+
+# How to start Project 
+
+ npm install 
+ 
+ npm run start 
+ 
+ open browser and run below url
+
```
Comment (jaimin45) [inline README.md:11]: hello jaimin

File: README.md [hunk 3]
```diff
@@ -1 +1,27 @@
-# demo-nodejs-tutorials
\ No newline at end of file
+# restful-api-node-express-crud
+simple project with node and express to create restful api's (CRUD)
+
+# How to start Project 
+
+ npm install 
+ 
+ npm run start
```
Comment (jaimin45) [inline README.md:8]: change the footer

File: README.md [hunk 4]
```diff
@@ -1 +1,27 @@
...
+ open browser and run below url
+ 
+ http://localhost:8000/
+
+# server.js 
+is used to handle all routes 
+
+## user.js 
+file is used for now as dummy database and helper file 
+we will modify the project one by one step.
```
Comment (jaimin45) [inline README.md:19]: change the spelling

File: swagger.json [hunk 5]
```diff
@@ -0,0 +1,190 @@
...
+      }
+    ],
+    "schemes": ["http"],
+    "consumes": ["application/json"],
+    "produces": ["application/json"],
+  
+
+  
+    "paths": {
+      "/users": {
```
Comment (kapilbarad) [inline swagger.json:23]: Can you please correct the paths of all the APIs? We're managing `tutorials` not `users`. Along with that please also correct `tag`, `summary` and other things which contains `user` keyword.

File: swagger.json [hunk 6]
```diff
@@ -0,0 +1,190 @@
...
+            }
+          }
+        }
+      }
+      
+
+    },
+
+    "definitions": {
+      "User": {
```
Comment (kapilbarad) [inline swagger.json:114]: Please design schema as I've requested in slack channel for only tutorials.

**Review by kapilbarad — CHANGES_REQUESTED** [review]

**Pull request merged**